pkg/service/replay/replay.go

```diff
@@ -220,6 +222,12 @@ func (r *Replayer) Start(ctx context.Context) error {
 			}
 		}
 
+		// check if its the last testset running -
+
+		if i == len(testSets)-1 {
```

Comment:
Please check if you ignore test set or select particular test set is this condition len(testSets)-1 is going to create any problem.